### PR TITLE
Improvement: Change modified information at BaseProcessDetails

### PR DESCRIPTION
**Pull request by lciolecki**
* Add modfiedBy field
* Add modifiedAt field
* Mark modificationDate as deprecated

### CODE REVIEW

**Review by arkadius — APPROVED** [review]
LGTM

File: ui/server/src/test/scala/pl/touk/nussknacker/ui/util/PdfExporterSpec.scala [hunk 1]
```diff
@@ -1,51 +1,48 @@
...
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now())
+      Comment(1L, "aa", details.processVersionId, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
```
Comment (mproch) [inline ui/server/src/test/scala/pl/touk/nussknacker/ui/util/PdfExporterSpec.scala:33]: Maybe we can make (1 to 29).map(_ => ...)?

**Review by mproch — APPROVED** [review]
:+1: I had small comment...

File: ui/server/src/test/scala/pl/touk/nussknacker/ui/util/PdfExporterSpec.scala (hunk 1 above)
Comment (lciolecki) [reply ui/server/src/test/scala/pl/touk/nussknacker/ui/util/PdfExporterSpec.scala:33]: Ups, I've not seen your comment.. ;)

**Pull request merged**